pkg/api/snapshot_resp.go

```diff
@@ -1,14 +1,18 @@
 package api
 
-import "github.com/pborman/uuid"
+import (
+	"time"
+
+	"github.com/pborman/uuid"
+)
 
 // SnapInfo contains static information about the snapshot.
 // Clients should NOT use this struct directly.
 type SnapInfo struct {
 	VolInfo       VolumeInfo `json:"snapinfo"`
 	ParentVolName string     `json:"parentname"`
 	Description   string     `json:"description"`
-	SnapTime      string     `json:"snaptime"`
+	CreatedAt     time.Time  `json:"createdAt"`
```

Comment:
Use `created-at` to be consistent with rest of the code and API.

Reply:
`createdAt` looks good, I have seen even in docker and kubernetes use `createdAt`

Reply:
done